### PR TITLE
Lots of little linting things

### PR DESCRIPTION
**Pull request by ErichDonGubler**
Feel totally free to reject any of this based on preference. :)

### CODE REVIEW

File: src/terminal.rs [hunk 1]
```diff
@@ -1,25 +1,20 @@
...
 use termion;
-
 use termrect::{RawPaintable, HasSize};
 use style::Style;
 use styledtext::StyledText;
 use std::io::Write;
 
 pub struct Terminal<W: Write> {
     size: (u32, u32),
-    current_style: Style,
```
Comment (ricochet1k) [inline src/terminal.rs]: I know it's technically unused, but I was leaving this there as a todo. Style diff and minimal updates is something I want to do soon.

File: src/terminal.rs [hunk 2]
```diff
@@ -1,25 +1,20 @@
...
     w: W,
 }
 
 impl<W: Write> Terminal<W> {
     pub fn new(mut w: W, size: (u32, u32)) -> Terminal<W> {
-        let current_style = Style::default();
-        write!(w, "{}", current_style).unwrap();
```
Comment (ricochet1k) [inline src/terminal.rs]: Same here. Please leave current_style alone for now.

**Review by ricochet1k — CHANGES_REQUESTED** [review]
Everything else looks good.

File: src/terminal.rs (hunk 2 above)
Comment (ErichDonGubler) [reply src/terminal.rs]: Done!

File: src/terminal.rs (hunk 1 above)
Comment (ErichDonGubler) [reply src/terminal.rs]: Done!

**Review by ricochet1k — APPROVED** [review]

**Pull request merged**